tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -241,7 +241,8 @@ const TSCONFIG = {
     'esModuleInterop': true,
     'allowSyntheticDefaultImports': true,
     'rootDir': '.',
-    'lib': ['esnext', 'dom', 'DOM.Iterable']
+    'lib': ['esnext', 'dom', 'DOM.Iterable'],
+    'noEmit': true,
```

Comment:
this was previously emitting the files on the disk, noEmit prevents that and the TSC validation still happens.